spec/lib/sms_api_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require "savon/mock/spec_helper"
+
+describe SMSApi do
+  let(:api) { SMSApi.new }
+
+  context "#sms_deliver" do
```

Comment:
```suggestion
  describe "#sms_deliver" do
```
`describe` is used when writing the classes/methods being tested; `context` is used when describing under which circumstances a method is being tested.